ci-operator/jobs/openshift/cincinnati-graph-data/openshift-cincinnati-graph-data-master-postsubmits.yaml

```diff
@@ -11,7 +11,7 @@ postsubmits:
         - push-to-quay
         command:
         - hack/graph-util.py
-        image: registry.access.redhat.com/ubi8/python-3.6
+        image: registry.access.redhat.com/ubi8/python-36
```

Comment:
Can we instead get closer to [the presubmit `publish` type][1]?  I don't like presubmit and postsubmit differing by more than a secret mount.

[1]: https://github.com/openshift/release/commit/1fe6e5c9af3d29132856182ff21fae7eabe33029#diff-306960387cb15167f7b1b69764025375R17